yogstation/code/modules/mob/living/simple_animal/drone/verbs.dm

```diff
@@ -0,0 +1,14 @@
+/////////////////////////
+//YOG DRONE VERBS//
+////////////////////////
+/mob/living/simple_animal/drone/verb/toggle_nightvision()
+	set name = "Toggle Nightvision"
+	set desc = "Toggles your ability to see in the dark."
+	set category = "Drone"
+	if (lighting_alpha == 140)
```

Comment:
use this see_invisible instead: https://github.com/yogstation13/yogstation/pull/2766/files#diff-c6ea51154efdb495c4af07bfe87dfca7R49